docs/source/keylime/index.rst

```diff
@@ -0,0 +1,135 @@
+Keylime
+=======
+
+ESI provides a guide to use `Keylime`_ outside Ironic for run-time attestation with `IMA`_.
+
+Prerequisites
+-------------
+* A private attestation network.
+* A node specified as Keylime server.
+* The nodes-to-be-attested with an active TPM 2.0 and keylime agent installed.
+
+Setup
+-----
+
+Build Keylime Server
+~~~~~~~~~~~~~~~~~~~~
+Provision the node:
+
+.. prompt:: bash $
+
+  metalsmith deploy --image <image> --network <network> --resource-class baremetal --ssh-public-key <path_to_public_key>
+
+* Note: the server needs to be connected to the public internet in order to download Keylime packages. Attaching the node to the public network could be as simple as running the command ``openstack esi node network attach (--network <network> | --port <port>) <node>``.
```

Comment:
This should go under "Install Keylime packages", since that's what this is for. It's a step before running git clone. Also, I'd just use the 'attach --network <public_network>' option, for simplicity.